src/dataflow/src/server.rs

```diff
@@ -108,6 +112,8 @@ pub enum SequencedCommand {
         id: GlobalId,
         /// The connector for the timestamped source.
         connector: SourceConnector,
+        /// Previously stored timestamp bindings.
+        bindings: Vec<(PartitionId, Timestamp, MzOffset)>,
```

Comment:
Similar to the sqlite question above: should we make this a more general struct so other sources can use it for this same purpose if needed? It appears pretty kafka specific now with partition and offset. I think we should either be explicit about its naming (put kafka somewhere in the various names), or make it more general.